### PR TITLE
Tiny doc fixes in performance dir

### PR DESCRIPTION
**Pull request by karmab**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Tiny doc fixes in performance dir

Fixes # (issue)

## Type of change

Please select the appropiate options:

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions: ocp4.3
- Hardware: 3 masters +2 workers in libvirt

## Checklist

- [ X] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@karmab Please add relevant ENV variables under https://github.com/openshift-kni/baremetal-deploy/blob/b332984b2a3a7fe28e660c25a5a7a0f8ba05dc34/features/performance/test/test_suite_test.go#L28

**Comment by karmab** [comment]
> @karmab Please add relevant ENV variables under
> 
> https://github.com/openshift-kni/baremetal-deploy/blob/b332984b2a3a7fe28e660c25a5a7a0f8ba05dc34/features/performance/test/test_suite_test.go#L28

done

File: features/performance/deploy.sh [hunk 1]
```diff
@@ -4,6 +4,14 @@ set -euo pipefail
 
 source $(dirname "$0")/../hack/common.sh
 
+# Check if there is at least one worker-rt node
+RTNODES=$(oc get node --selector=node-role.kubernetes.io/worker-rt="")
+if [ "${RTNODES}" == "" ; then
```
Comment (cynepco3hahue) [inline features/performance/deploy.sh]: you forgot `]`

Comment (karmab) [reply features/performance/deploy.sh]: fixed!

**Pull request merged**